ED/src/init/ed_params.f90

```diff
@@ -2483,17 +2483,17 @@ subroutine init_pft_alloc_params()
 
    !----- Carbon-to-biomass ratio of plant tissues. ---------------------------------------!
    C2B    = 2.0
-   !---------------------------------------------------------------------------------------! 
+   !---------------------------------------------------------------------------------------!
 
    !---------------------------------------------------------------------------------------!
    !     Wood density.  Currently only tropical PFTs need it.  C3 grass density will be    !
    ! used only for branch area purposes.                                                   !
    !---------------------------------------------------------------------------------------!
    !---- [KIM] new tropical parameters. ---------------------------------------------------!
-   rho(1)     = 0.40   ! 0.20
-   rho(2)     = 0.40   ! 0.53
-   rho(3)     = 0.60   ! 0.71
-   rho(4)     = 0.87   ! 0.90
+   rho(1)     = 0.20   ! 0.40
```

Comment:
When proposing new default parameters, please be more descriptive in your commits about why things are being changed and where the numbers are coming from.

Reply:
If you check your repo before PR #200 you will see that what this commit does is only to revert to how the numbers were originally. Around line 2490 there was a comment saying 
`[KIM] new tropical parameters.`
and commented next to the values where the numbers you saw.
So I thought these were more updated parameters. Later on, talking with Marcos, I realized that the new parameters where the one actually being used and I consequently updated the PR #200. Since this present PR was made before amending the change, it appears as if there is a difference (not sure why this is happening though).

Reply:
@crollinson, I had noticed in the first PR from @manfredo89 that the default values for rho were different from the previous mainline, and I asked him to revert to the original ones.  The numbers in this PR are the original numbers.

Reply:
The strange thing is that also the numbers in PR#200 are correct. I guess this appears as a change because my ed_params had the other values in before commit 6a13a18048bb00aa7bdcbe1b52836f767f26c3fb (https://github.com/EDmodel/ED2/pull/200/commits/6a13a18048bb00aa7bdcbe1b52836f767f26c3fb)